src/include/OpenImageIO/simd.h

```diff
@@ -634,6 +634,34 @@ class int4 {
 #endif
     }
 
+    OIIO_FORCEINLINE void load (const int *values, int n) {
+#if defined(OIIO_SIMD_SSE)
+        switch (n) {
+        case 1:
+            m_vec = _mm_castps_si128 (_mm_load_ss ((const float *)values));
```

Comment:
Oh, and that's not what you meant at all, is it? You meant the cast of int* to float*. 
I'll double check on gcc. Sorry for all the spam.